plugins/index.js

```diff
@@ -34,5 +82,6 @@ function withSceneDelegate(config, action) {
   });
 }
 
+exports.macos = macos;
```

Comment:
On second thought, maybe it makes more sense to move these to a separate file so the import becomes `react-native-test-app/plugins/macos`.